Doc/library/tempfile.rst

```diff
@@ -219,11 +219,14 @@ The module defines the following user-callable items:
       * On all other platforms, the directories :file:`/tmp`, :file:`/var/tmp`, and
         :file:`/usr/tmp`, in that order.
 
-   #. As a last resort, the current working directory.
+   #. The current working directory.
 
    The result of this search is cached, see the description of
    :data:`tempdir` below.
 
+   :exc:`FileNotFoundError` is raised if no suitable directory can be found.
+
```

Comment:
Just combine the above two sentences:

> If no usable directory can be found :exc:`FileNotFoundError` is raised.  The result ...

Note there are two spaces between them. :-)